kt/kt.go

```diff
@@ -206,7 +208,56 @@ func NewConnTLS(host string, port int, poolsize int, timeout time.Duration, cred
 
 // NewConn creates a connection to an Kyoto Tycoon endpoint.
 func NewConn(host string, port int, poolsize int, timeout time.Duration) (*Conn, error) {
-	return newConn(host, port, poolsize, timeout, "")
+	parts := strings.Split(host, "://")
```

Comment:
As we use NewTLSConn() to create connection it may be useful to implement logic for it too.